staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -239,7 +239,7 @@ func (f *fancyResponseWriterDelegator) Hijack() (net.Conn, *bufio.ReadWriter, er
 	// fake a response status before protocol switch happens
 	f.processCode(http.StatusSwitchingProtocols)
 
-	// This will be ignored if WriteHeader() function has aready been called.
+	// This will be ignored if WriteHeader() function has already been called.
 	// It's not guaranteed Audit-ID http header is sent for all requests.
 	// For example, when user run "kubectl exec", apiserver uses a proxy handler
```

Comment:
when _a_ user _runs_...

Reply:
@xianlubird  This is still not addressed.

Reply:
@dixudx @nikhita Done